doc/developer-notes.md

```diff
@@ -91,6 +91,9 @@ code.
   - Compile-time constant names are all uppercase, and use `_` to separate words.
   - Class names, function names, and method names are UpperCamelCase
     (PascalCase). Do not prefix class names with `C`.
+  - Enumerators may be snake_case, PascalCase, or ALL_CAPS. This is a more tolerant policy than
```

Comment:
In commit "doc: update enum naming in developer notes" (2d813c42ed5573d3afd80d677e2ded41e8d5b7e8)

Suggestion to s/Enumerators/Enumerator constants/ to be clear this is referring to enum constant names not enum type names (maybe this is clear to others, but I had to look it up)

Reply:
done